sdk/messaging/azeventhubs/consumer_client.go

```diff
@@ -20,6 +20,9 @@ type ConsumerClientOptions struct {
 	// ApplicationID is used as the identifier when setting the User-Agent property.
 	ApplicationID string
 
+	// A custom endpoint address that can be used when establishing the connection to the service.
+	CustomEndpoint string
```

Comment:
Is this name consistent with other languages?

Reply:
It's shorter - the other languages have gone with `CustomEndpointAddress` but our SDKs tend to just use 'endpoint' as the terminology for the same thing.

Reply:
(talked offline with Rick, I'm going to leave this one as-is as)